microbial_thermodynamics/cell_growth_ODE.py

```diff
@@ -0,0 +1,143 @@
+"""Summary:
+This calculates rates of change of population growth, ribosome fraction and
+internal energy production.
+
+Desctiption:
+I am uncertain if the R in calculate_lam and the r in dr are the same thing (ribosome
+fraction). Should I find values for gam and R or write functions.
+"""  # noqa: D205
+
+from typing import Callable
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def calculate_r_star(a: float, om: float = 1e9, Q: float = 0.45) -> float:
+    """Calculates ideal ribosome fraction.
+
+    Args:
+       a: internal energy (ATP) concentration
+       om: Ribosome fraction half saturation constant
+       Q: housekeeping fraction
+    """
+    u = (a * (1 - Q)) / (a + om)
+    return u
+
+
+def calculate_lam(gam: float, R: float, m: float = 1e8, fb: float = 0.7) -> float:
+    """Calculates species growth.
+
+    Args:
+     gam: Effective translation elongation rate
```

Comment:
gam is equation (6) in the [original paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009643), so you should make this a function that gets called